website2/src/views/Forum/AboutPage.tsx

```
@@ -5,82 +5,90 @@ import { Divider, NoData } from '@/components/ui';
-        {/* Introduction */}
-        <div
-          dangerouslySetInnerHTML={{
-            __html: renderContent(data.introduction),
-          }}
-        />
+        {/* Introduction Section (kept out of the redesign) */}
+        <section className="space-y-6">
+          <h2 className="text-2xl font-bold text-left">Introduction</h2>
+          <div
+            className="prose"
+            dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)